src/aap_eda/settings/default.py

```diff
@@ -21,17 +23,20 @@
 
 from .post_load import post_loading
 
-EDA_SETTINGS_FILE = "/etc/eda/settings.yaml"
+EDA_SETTINGS_FILE = os.environ.get(
```

Comment:
I'm missing something here. why we call directly the os.environ? it should not be handled by dynaconf?

Reply:
This is before we call dynaconf to import settings. We ask dynaconf to load additional settings from this file. It was hardcoded. Now can be changed by env, also good for testing.